src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14.06px;
+  text-align: left;
+
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  padding-left: 50px;
+  padding-right: 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__list {
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). Instead of styling the '.nav__list' by removing the default list style, create a class for the 'ul' element and style that class.